src/control/server/engine/config.go

```diff
@@ -690,3 +767,26 @@ func (c *Config) WithStorageIndex(i uint32) *Config {
 	c.Storage.EngineIdx = uint(i)
 	return c
 }
+
+// WithEnvVarAbtThreadStackSize sets environment variable ABT_THREAD_STACKSIZE.
+func (c *Config) WithEnvVarAbtThreadStackSize(stack_size uint16) *Config {
+	return c.WithEnvVars(fmt.Sprintf("ABT_THREAD_STACKSIZE=%d", stack_size))
+}
+
+// WithEnvVarPMemObjSdsAtCreate sets PMEMOBJ_CONF env. var. to sds.at_create=0/1 value
+func (c *Config) WithEnvVarPMemObjSdsAtCreate(value uint8) *Config {
+
+	pmemobjConfStr, pmemobjConfErr := c.GetEnvVar("PMEMOBJ_CONF")
+	if pmemobjConfErr != nil {
+		return c.WithEnvVars(fmt.Sprintf("PMEMOBJ_CONF=sds.at_create=%d", value))
+	} else {
+		if strings.Contains(pmemobjConfStr, "sds.at_create") {
+			var re = regexp.MustCompile(`(;?sds.at_create=[01])`)
```

Comment:
```suggestion
			re := regexp.MustCompile(`(;?sds.at_create=[01])`)
```
A small suggestion for making this a bit more idiomatic in Go.